templates/outer-loop/Dockerfile

```diff
@@ -40,19 +40,30 @@ COPY --from=compile --chown=1001:0 /work/configlibdir/ /config
 # 2c) Copy user defined server config, bootstrap.properties, etc.
 COPY --from=compile --chown=1001:0 /work/config/ /config/
 
-# 2d) Install the required features based on the defined configuration. This enables Liberty features and image size to be fit-for-purpose using featureUtility.
-RUN features.sh
-
-# 2e) Add the microprofile health feature configuration if it is not already defined in the user's configuration.
+# 2d) Add the microprofile health feature configuration if it is not already defined in the user's configuration.
+#     This allows k8s to use the deployment's health probes.
 RUN mkdir -p /tmp/stack/config/configDropins/defaults
 COPY --from=compile --chown=1001:0 /stack/ol/config/configDropins/defaults/ /tmp/stack/config/configDropins/defaults/
-RUN if ! /opt/ol/wlp/bin/productInfo featureInfo | grep -q mpHealth-[0-9]*.[0-9]*; then \
-        cp /tmp/stack/config/configDropins/defaults/liberty-stack-mpHealth.xml /config/configDropins/defaults; \
-        features.sh; \
+
+ARG ADD_MP_HEALTH=true
+RUN if [ "$ADD_MP_HEALTH" = "true" ]; then \
+        /opt/ol/wlp/bin/server start; \
+        /opt/ol/wlp/bin/server stop; \
+        if ! grep "The server installed the following features" /logs/messages.log | grep -q 'mpHealth-[0-9]*.[0-9]*\|microProfile-[0-9]*.[0-9]*'; then \
+            echo "Missing mpHealth feature - adding config snippet"; \
+            cp /tmp/stack/config/configDropins/defaults/liberty-stack-mpHealth.xml /config/configDropins/defaults; \
+        fi; \
+        rm -f /logs/*; \
+    elif [ "$ADD_MP_HEALTH" != "false" ]; then \
+        echo "Invalid ADD_MP_HEALTH value: $ADD_MP_HEALTH. Valid values are \"true\" | \"false\" "; \
     fi
+
 RUN rm -rf /tmp/stack
 
-# 2f) Copy the application binary
+# 2e) Copy the application binary
 COPY --from=compile --chown=1001:0 /work/outer-loop-app/target/*.[ew]ar /config/apps
+
+# 2f) Run configure.sh
+ENV OPENJ9_SCC=true
```

Comment:
I mentioned this could be a follow-up issue so that's OK, but noting this doesn't allow turning off SCC as arg.

I kind of like creating a new arg:  QUICK_BUILD
and using it to both disable:   ADD_MP_HEALTH and disable the SCC though I could see other opinions.

Is there an issue open for this ? (Could you check?)  Thanks.

Reply:
I think https://github.com/OpenLiberty/application-stack/issues/23 could cover this.